File: articles/proxy-variables-and-measurement-error/haskell/ProxyMeasurementRisk.hs
module Main where

main :: IO ()
main = do
  let validityGap = 0.42
      missingness = 0.12
      differentialError = 0.24
      labelError = 0.08
      score = (validityGap + missingness + differentialError + labelError) / 4
  putStrLn ("measurement_risk_score=" ++ show score)

File: articles/proxy-variables-and-measurement-error/rust/proxy_measurement_risk.rs
fn main() {
    let validity_gap = 0.42;
    let missingness = 0.12;
    let differential_error = 0.24;
    let label_error = 0.08;
    let score = (validity_gap + missingness + differential_error + label_error) / 4.0;
    println!("measurement_risk_score={:.4}", score);
}
